Create ui-c8y-1021-8-0-provide-option-to-enable-disable-dashboard-manager-in-the-app-configuration.md (#2676)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1021-8-0-provide-option-to-enable-disable-dashboard-manager-in-the-app-configuration.md

---
date: ""

> title: Option to enable or disable Dashboard Manager is added to Cockpit Application configuration

Reviewer: ```suggestion
title: Option added to enable or disable the dashboard manager in the application configuration
```

> title: Provide option to enable/disable Dashboard Manager in the App configuration (#7567) [GRAFT][release/cd] (#7640)

Author: The Claude AI suggested to use the following title:
```suggestion
title: Option to enable or disable Dashboard Manager in App configuration
```

> title: Provide option to enable/disable Dashboard Manager in the App configuration (#7567) [GRAFT][release/cd] (#7640)

Reviewer 2: ```suggestion
title: Option to enable or disable Dashboard Manager is added to Cockpit Application configuration
```

product_area: Application enablement & solutions
change_type:
  - value: change-QHu1GdukP
    label: Feature
component:
  - value: component-YdSEScrEC
    label: Cockpit
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-61177
version: 1021.8.0
---

> Provide option to enable/disable Dashboard Manager in the App configuration (#7567) [GRAFT][release/cd] (#7640)

Author: The Claude AI suggested to use the following body:
```suggestion
To provide more flexibility for users, an option has been added to the App configuration which allows enabling or disabling the Dashboard Manager. Previously, the Dashboard Manager was always enabled without the possibility to turn it off. With this change, users can now decide whether they want to use the Dashboard Manager functionality in their applications or not, depending on their specific requirements. This configuration option impacts all applications where the Dashboard Manager was previously available and used.
```

> Provide option to enable/disable Dashboard Manager in the App configuration (#7567) [GRAFT][release/cd] (#7640)

Reviewer 2: User is now able to disable or enable Dashboard manager from Application configuration in Cockpit Configuration. By default, Dashboard manager is enabled.

> Provide option to enable/disable Dashboard Manager in the App configuration (#7567) [GRAFT][release/cd] (#7640)

Reviewer 2: ```suggestion
User is now able to disable or enable Dashboard manager from Application configuration in Cockpit Configuration. By default, Dashboard manager is enabled.
```

> User is now able to disable or enable Dashboard manager from Application configuration in Cockpit Configuration. By default, Dashboard manager is enabled.

Reviewer: ```suggestion
In the Cockpit application, it is now possible to  disable or enable the dashboard manager in the **Application configuration** page. By default, the dashboard manager is enabled.
```
